atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/ColumnMetadataDescription.java

```diff
@@ -48,6 +50,25 @@ public ColumnMetadataDescription(DynamicColumnDescription dynamicColumn) {
         this.dynamicColumn = dynamicColumn;
     }
 
+    public static ColumnMetadataDescription singleNamed(String shortName, String longName, ValueType valueType) {
```

Comment:
nit, probably later if you're feeling bothered, it may be nice to have these on the builder? 

so you have something on the builder so you can do this:
```java
TableMetadata.internal()
  .namedColumn("shortName", "longName", ValueType.BLOB)
  .dynamicColumn("name", ValueType.BLOB, ValueType.SHA256)
  // etc
```

```java
class Builder extends ImmutableBuilder {
  public Builder namedColumn(String name, String longName, ValueType valueType) {
    // contents on the method here
  }
}
```